### PR TITLE
Update deprecation policy for metrics

### PR DESCRIPTION
**Pull request by abhide**
Tickets: #2554
Signed-off-by: Amey Bhide <amey15@gmail.com>

### CODE REVIEW

**Review by youngnick — APPROVED** [review]

**Review by stevesloka — APPROVED** [review]

File: site/_resources/deprecation-policy.md [hunk 1]
```diff
@@ -61,3 +61,13 @@ We use the following cycle for config file settings:
...
 - The setting is moved or removed.
 
+## Metrics
+
+Removing/Renaming metrics is a breaking change as it might cause alerts/ dashboards to break for operators.
+
+We try to use following cycle for deprecating metrics:
+- Description for metrics should be updated to mark these metrics as deprecated.
+- If you are renaming metrics, then add metrics with new names
+- Update [grafana dashboards](../../examples/grafana/02-grafana-configmap.yaml) if applicable
```
Comment (stevesloka) [inline site/_resources/deprecation-policy.md:71]: The links to the dashboards & alertmanager aren't working. Here's the preview page: https://deploy-preview-3362--projectcontour.netlify.app/resources/deprecation-policy/

**Review by stevesloka — CHANGES_REQUESTED** [review]

File: site/_resources/deprecation-policy.md (hunk 1 above)
Comment (abhide) [reply site/_resources/deprecation-policy.md]: fixed.

File: site/_resources/deprecation-policy.md [hunk 2]
```diff
@@ -61,3 +61,13 @@ We use the following cycle for config file settings:
 - We wait the timeline period
 - The setting is moved or removed.
 
+## Metrics
+
+Removing/Renaming metrics is a breaking change as it might cause alerts/ dashboards to break for operators.
+
+We try to use following cycle for deprecating metrics:
```
Comment (sunjayBhatia) [inline site/_resources/deprecation-policy.md:68]:
```suggestion
We try to use the following cycle for deprecating metrics:
```

Comment (sunjayBhatia) [reply site/_resources/deprecation-policy.md:68]: Just a tiny nit!

**Review by sunjayBhatia — COMMENTED** [review]
Looks good just a tiny suggested change

File: site/_resources/deprecation-policy.md (hunk 2 above)
Comment (abhide) [reply site/_resources/deprecation-policy.md]: done

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**